### DISCUSSION TITLE
I am not able to install package armadillo I have visual studio 2017 installed

### DISCUSSION THREAD

**Discussion by rameshdk12** [microsoft/vcpkg#38300] [Q&A]
I get the following output on trying to install armadillo:

Computing installation plan...
The following packages will be built and installed:
    armadillo[core]:x86-windows
  * lapack[core]:x86-windows
  * lapack-reference[blas-select,core,noblas]:x86-windows
  * vcpkg-gfortran[core]:x86-windows
Additional packages (*) will be modified to complete this operation.
Detecting compiler hash for triplet x86-windows...
Starting package 1/4: vcpkg-gfortran:x86-windows
Building package vcpkg-gfortran[core]:x86-windows...
Could not locate cached archive: C:\Users\Ramesh\AppData\Local\vcpkg\archives\02\0277d0244948fac3319735c06abe06aa36e97873.zip
-- The Fortran compiler identification is unknown
-- No Fortran compiler found on the PATH. Using MinGW gfortran!
-- Downloading https://repo.msys2.org/mingw/i686/mingw-w64-i686-gcc-fortran-10.2.0-1-any.pkg.tar.zst...
-- Downloading https://repo.msys2.org/mingw/i686/mingw-w64-i686-gcc-fortran-10.2.0-1-any.pkg.tar.zst... Failed. Status: 22;"HTTP response code said error"
CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:181 (message):

      Failed to download file.
      If you use a proxy, please set the HTTPS_PROXY and HTTP_PROXY environment
      variables to "https://user:password@your-proxy-ip-address:port/".
      Otherwise, please submit an issue at https://github.com/Microsoft/vcpkg/issues

Call Stack (most recent call first):
  scripts/cmake/vcpkg_acquire_msys.cmake:114 (vcpkg_download_distfile)
  scripts/cmake/vcpkg_find_fortran.cmake:29 (vcpkg_acquire_msys)
  ports/vcpkg-gfortran/portfile.cmake:3 (vcpkg_find_fortran)
  scripts/ports.cmake:79 (include)

Error: Building package vcpkg-gfortran:x86-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: vcpkg-gfortran:x86-windows
  Vcpkg version: 2020.06.15-nohash

**Comment by autoantwort** [comment]
> Vcpkg version: 2020.06.15-nohash

Please update your vcpkg version: `git pull` + `./bootstrap-vcpkg.bat`
